microsoft/terminal
# #1536 - Feature Request - Duplicate tab

Added keybinding option to duplicate focused tab
## Summary of the Pull Request
Added necessary delcarations and implmentations to provide a new keybinding for duplicating the focused tab.

## References

## PR Checklist
* [x] Closes #1536 
* [ ] Tests <del>added/passed</del> N/A
* [x] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #1536  

## Detailed Description of the Pull Request / Additional comments
As discussed in this issue, a simple implementation for duplicating the focused tab was made by me.
I added the needed declarations for the new keybinding and added a method to duplicate the focused tab in `App.cpp`.

## Validation Steps Performed
* Add keybinding element to `profiles.json`
	* `"command" : "duplicateTab"`
	* `"keys": [ "ctrl+shift+d" ]`
* Open Terminal
* Verify other features and functionalities work as usual without unexpected behaviour
* Use configured keybinding to validate the duplication of the focused tab
